src/polycubed/src/utils/veth_peer.h

```diff
@@ -0,0 +1,70 @@
+#include <stdlib.h>
+#include <vector>
+#include <sstream>
+#include <iostream>
+#include <stdexcept>
+#include <thread>
+#include <sys/ioctl.h>
+#include <linux/limits.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <net/if.h>
+#include <arpa/inet.h>
+#include <netinet/ether.h>
+#include <netlink/socket.h>
+#include <netlink/netlink.h>
+#include <netlink/route/link.h>
+#include <netlink/route/link/veth.h>
+
+#define NLMSG_TAIL(nmsg) \
+        ((struct rtattr *) (((char *) (nmsg)) + NLMSG_ALIGN((nmsg)->nlmsg_len)))
+
+struct nl_req {
+	struct nlmsghdr nlmsg;
+	struct ifinfomsg ifinfomsg;
+};
+
+struct nl_ipreq {
+	struct nlmsghdr nlmsg;
+	struct ifaddrmsg ifaddrmsg;
+};
+
```

Comment:
Are those structure  and that define needed in this header?, I think those can be moved to the cpp.